packages/protocol/contracts/rewards/StakingRewards.sol

```diff
@@ -30,9 +30,18 @@ contract StakingRewards is ERC721PresetMinterPauserAutoIdUpgradeSafe, Reentrancy
     TwentyFourMonths
   }
 
+  enum StakedPositionType {
+    Fidu,
+    CurveLP
+  }
+
   struct StakedPosition {
+    // @notice Type of the staked position
```

Comment:
Believe we need to add these to the end of the struct definition, otherwise the storage layout of the struct would be corrupted for existing positions

Reply:
Ah yes, thanks! Fixed in latest commits.